### ISSUE TITLE
serviceName attribute from @Webservice not used

### ISSUE THREAD

**Issue by querdenker2k** [quarkiverse/quarkus-cxf#439]
When i have a separate api and impl project the generated wsdl is incorrect.
Inside the api the classes from the wsdl are generated.
Inside the impl project the api is used to implement the service.
Quarkus version: 2.8.3
Cxf version: 1.1.0

The ?wsdl result is
```
<wsdl:definitions xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:wsdl="http://schemas.xmlsoap.org/wsdl/" xmlns:tns="http://tempuri.org/" xmlns:soap12="http://schemas.xmlsoap.org/wsdl/soap12/" xmlns:ns1="http://schemas.xmlsoap.org/soap/http" name="Calculator" targetNamespace="http://tempuri.org/">
<wsdl:portType name="CalculatorWebServiceImpl_ClientProxy"> </wsdl:portType>
<wsdl:binding name="CalculatorSoapBinding" type="tns:CalculatorWebServiceImpl_ClientProxy">
<soap12:binding style="document" transport="http://schemas.xmlsoap.org/soap/http"/>
</wsdl:binding>
<wsdl:service name="Calculator">
<wsdl:port binding="tns:CalculatorSoapBinding" name="CalculatorWebServiceImpl_ClientProxyPort">
<soap12:address location="http://localhost:8080/ws/calculator"/>
</wsdl:port>
</wsdl:service>
</wsdl:definitions>
```
Here is a reproducer: https://github.com/querdenker2k/cxf-servicename-reproducer

With Cxf version 0.9 is it still working but then the serviceName is wrong: **CalculatorWebServiceService** instead of **Calculator**

**Comment by shumonsharif** [comment]
Hi @querdenker2k Can you post an example of what the WSDL _should_ look like?  My initial guess is that it may be a [Jandex index](https://quarkus.io/guides/cdi-reference#how-to-generate-a-jandex-index) issue in your setup, though I did not go through your code.

**Comment by querdenker2k** [comment]
It's the unmodified wsdl from your repo from the tests

**Comment by shumonsharif** [comment]
@querdenker2k Is there a reason you have this line?  I believe that may be the root cause of your issue, it's not quite a valid implementation class with that annotation.

https://github.com/querdenker2k/cxf-servicename-reproducer/blob/main/impl/src/main/java/io/quarkus/cxf/reproducer/impl/CalculatorWebServiceImpl.java#L8

**Comment by querdenker2k** [comment]
ahh nice, i didn't know that this occurs in such an error.
but in fact i have a further problem for which i wrote the reproducer.
the @WebService#serviceName annotation was not read.
Instead of **Calculator** the service name inside the generated wsdl is **CalculatorWebServiceService**

**Comment by shumonsharif** [comment]
Yes, that appears to be a valid issue.  Will try to have it addressed in the next release.  Thanks!

**Comment by querdenker2k** [comment]
@shumonsharif can you say when a new release will be created

**Comment by shumonsharif** [comment]
Hey @querdenker2k Apologies, I wasn't sure if you needed this quickly -- otherwise I could have given you a patch release a while back.  I'm going to do a release in the next few hours.

**Comment by querdenker2k** [comment]
i am waiting for https://github.com/quarkusio/quarkus/issues/25325 as well. but i hope the next release will come in a few days.

**Comment by shumonsharif** [comment]
Just released quarkus-cxf 1.2.0.

> i am waiting for [quarkusio/quarkus#25325](https://github.com/quarkusio/quarkus/issues/25325) as well. but i hope the next release will come in a few days.

This issue should already be fixed in Quarkus 2.10.

Can you please test and let me know if everything is working as expected?

**Comment by querdenker2k** [comment]
Yes, thanks a lot, its working as expected. But Quarkus 2.10 hasn't been released yet.
